pkg/proto/v0/settings.pb.micro_test.go

```diff
@@ -1520,16 +1520,26 @@ func TestListGetBundleSettingMixedPermission(t *testing.T) {
 			defer teardown()
 
 			// create bundle with the defined settings
-			bundle := &bundleStub
```

Comment:
note this used to change the original bundleStub, so the following tests would not be able to rely on the original state. Better to create a new bundle